app/models/accounting/qb/transaction_extractor.rb

```diff
@@ -49,6 +49,7 @@ def extract_line_items
         txn.quickbooks_data['line_items'].each do |li|
           begin
             acct = Accounting::Account.find_by(qb_id: li[qb_li_detail_key]['account_ref']['value'])
+            raise StandardError "unprocessable_account" if acct.nil?
```

Comment:
11412 change. We raise error if acct nil so its caught in the rescue block and a ProblemLoanTxn is made. Otherwise the user sees an PG NOT NULL error on the loan, but doesn't know what txn caused the issue.